graphql/src/execution/ast.rs

```diff
@@ -351,7 +350,7 @@ pub(crate) fn resolve_object_types(
         .ok_or_else(|| QueryExecutionError::AbstractTypeError(name.to_string()))?
     {
         s::TypeDefinition::Interface(intf) => {
-            for obj_ty in &schema.types_for_interface()[&EntityType::new(intf.name.to_string())] {
+            for obj_ty in &schema.types_for_interface()[&intf.name] {
```

Comment:
Is there some risk that this could cause a panic?

Reply:
In theory, it could, though we've had this there for a long time w/o issues, and the changes in the code make it less likely to panic.